readme.md

```diff
@@ -213,6 +213,19 @@ Default: `process.cwd()`
 
 Current working directory of the project to retrieve the diagnostics for.
 
+##### typingsFile
+
+Type: `string`<br>
+Default: `''`
+
+Path to the type definitions of the project.
```

Comment:
It's not clear to the user (or at least me) when someone would want to set this. And is the default really `''`?

Reply:
I've made some changes, explained what the default is. Let me know if that's good or needs any further change :)

Reply:
That doesn't resolve my concern though. I'm asking to document (in the readme) the use-case for the option.

Reply:
Ah, gotcha. Will work on that 😉

Reply:
So I guess this is the last thing we should work out?

@sindresorhus Do you mean something like this perhaps?

> Path to the type definition file you want to test. This can be useful when using a test runner to test specific type definitions per test.

Reply:
👍🏻

Reply:
@smartclash If you can update the description I will merge it in :).